Pod/Classes/RCAnalytics.swift

```diff
@@ -39,14 +43,26 @@ public class RCAnalytics: NSObject {
     }
     
     public func launch() {
-        return track("appLaunch");
+        if (userDefaults.boolForKey(HAS_LAUNCHED_KEY) == false) {
+            track("appInstall");
+        }
+
+        track("appLaunch");
+
+        userDefaults.setBool(true, forKey: HAS_LAUNCHED_KEY);
+        userDefaults.synchronize();
```

Comment:
As per [this SO answer](http://stackoverflow.com/questions/9964371/how-to-detect-first-time-app-launch-on-an-iphone).